app/views/playbook/about.html.slim

```diff
@@ -0,0 +1,155 @@
+                  and usernames. Physical attributes such as fingerprints and DNA can also be
+                  part of an identity record.
+              h4.mt0.mb1.h3 How does identity and access management work?
+              p
+                | System administrators assign access privileges to each identity record. These
+                  privileges authorize certain activities and forbid others. To “open the door”
+                  safely, however, administrators need confidence that the users knocking at the
+                  door are who they say they are.
+              p.mb3
+                | To give the system and its administrators confidence in their identities, users
+                  need to prove their identities through an activity called authentication. Users
+                  authenticate themselves by presenting evidence linking themselves to records.
+                  To do that, users first help the system validate their record — for example,
+                  by typing in a username. Then users hand over the evidence — often, passwords
+                  or other information only the real person would know.
+              h4.mt0.mb1.h3 What does having an identity record enable?
+              p.mb3
+                | Identity systems don’t just benefit system administrators. With an authenticated
+                  digital identity, users can do some very handy things. Here’s a small list:
+              ul.mt0.mb6.ml1.pl1.pb-list--bullet
+                li.mb2.gray
+                  span.bold
+                    | Pre-filling online forms with verified information speeds up application
+                      processing.
+                  |  There’s less redundant effort, and users don’t need to worry about basic
+                    errors.
+                li.mb2.gray
+                  span.bold
+                    | Authenticated users can access and download data the system holds about
+                      them, such as account activity.
+                  |  With a verified legal identity, the user can access very sensitive medical
+                    or financial records and even download them.
+                li.mb2.gray
+                  span.bold
+                    | Identity systems can protect your privacy.
+                  |  If you need to be 21 or older to access a service, you can authorize an
+                    identity system to confirm your age without sharing your exact birth date.
+              h3.mb2.h1 Implementation
+              = image_tag(asset_url('playbook/hr-red-6.svg'), alt: 'hr', class: 'mb3')
+              p.mb6 <a href="#">Please read the developer documentation.</a>
+              h3.mb2.h1 Resources
+              = image_tag(asset_url('playbook/hr-red-7.svg'), alt: 'hr', class: 'mb3')
+              p <a href="#">National Institute of Standards in Technology</a> (NIST 800-63-3)
```

Comment:
semantically, this should be a list with the `list-reset` class to remove bullets, padding